src/index.ts

```diff
@@ -37,9 +37,9 @@ export default class HobbitsP2PNetwork {
   /**
    * Connects to static peers
    */
-  private connectStaticPeers = () => {
-    this.bootnodes.map((bootnode: string) => {
-      this.connect(bootnode);
+  private connectStaticPeers = async () => {
+    await this.bootnodes.map(async (bootnode: string) => {
```

Comment:
```javascript
await Promise.all(this.bootnodes.map....);
```
Currently you have `await [promise, promise]` which won't wait for all promises inside array to resolve.

Reply:
*I hate promises* *lets use GO*

Reply:
lol